diff --git a/Makefile.pre.in b/Makefile.pre.in
index 58178d2..a59696a 100644
--- a/Makefile.pre.in
+++ b/Makefile.pre.in
@@ -1218,10 +1218,10 @@ inclinstall:
 
 # Install the library and miscellaneous stuff needed for extending/embedding
 # This goes into $(exec_prefix)$(DEBUG_EXT)
-LIBPL=		$(LIBP)/config-$(MULTIARCH)$(DEBUG_EXT)
+LIBPL=		$(LIBP)/config
 
 # pkgconfig directory
-LIBPC=		$(LIBDIR)/$(MULTIARCH)/pkgconfig
+LIBPC=		$(LIBDIR)/pkgconfig
 
 libainstall:	@DEF_MAKE_RULE@ python-config
 	@for i in $(LIBDIR) $(LIBP) $(LIBPL) $(LIBPC); \
diff --git a/Misc/python-config.sh.in b/Misc/python-config.sh.in
index a69077b..5cc3e4a 100755
--- a/Misc/python-config.sh.in
+++ b/Misc/python-config.sh.in
@@ -43,7 +43,7 @@ LINKFORSHARED="@LINKFORSHARED@"
 OPT="@OPT@"
 PY_ENABLE_SHARED="@PY_ENABLE_SHARED@"
 LIBDEST=${prefix}/lib/python${VERSION}
-LIBPL=${LIBDEST}/config-${MULTIARCH}${ABIFLAGS}
+LIBPL=${LIBDEST}/config
 SO="${ABIFLAGS}.so"
 PYTHONFRAMEWORK="@PYTHONFRAMEWORK@"
 INCDIR="-I$includedir/python${VERSION}${ABIFLAGS}"
