diff --git a/src/killall5.c b/src/killall5.c
index 545b8e3..f65db7c 100644
--- a/src/killall5.c
+++ b/src/killall5.c
@@ -966,9 +966,9 @@ int matches(PROC *o, PROC *p)
 	char *oargv1, *pargv1;
 	if ((o->argv0 && p->argv0 && !strcmp(o->argv0,p->argv0))) {
 		if (o->argv1 && p->argv1) {
-			if ((oargv1 = canonicalize_file_name(o->argv1)) == NULL)
+			if ((oargv1 = realpath(o->argv1, NULL)) == NULL)
 				oargv1 = strdup(o->argv1);
-			if ((pargv1 = canonicalize_file_name(p->argv1)) == NULL)
+			if ((pargv1 = realpath(p->argv1, NULL)) == NULL)
 				pargv1 = strdup(p->argv1);
 			if (! strcmp(oargv1, pargv1)) {
 				ret = 1;
diff --git a/src/mountpoint.c b/src/mountpoint.c
index 3daf470..f99d498 100644
--- a/src/mountpoint.c
+++ b/src/mountpoint.c
@@ -23,6 +23,7 @@
  *		Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
  */
 
+#include <sys/types.h>
 #include <sys/stat.h>
 #include <unistd.h>
 #include <stdlib.h>
