From cac5ca4a216105c9ef82cc7e779ba9560bb2e198 Mon Sep 17 00:00:00 2001
From: Alexey Brodkin <abrodkin@synopsys.com>
Date: Thu, 13 Sep 2018 17:13:20 +0300
Subject: [PATCH 2/2] icu: Add ARC support

Signed-off-by: Alexey Brodkin <abrodkin@synopsys.com>

Upstream-Status: Submitted [ https://github.com/unicode-org/icu/pull/149 ]
---
 source/i18n/double-conversion-utils.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/source/i18n/double-conversion-utils.h b/source/i18n/double-conversion-utils.h
index 57fc49b..0bd3e83 100644
--- a/source/i18n/double-conversion-utils.h
+++ b/source/i18n/double-conversion-utils.h
@@ -86,7 +86,7 @@ inline void abort_noreturn() { abort(); }
     defined(__SH4__) || defined(__alpha__) || \
     defined(_MIPS_ARCH_MIPS32R2) || \
     defined(__AARCH64EL__) || defined(__aarch64__) || \
-    defined(__riscv)
+    defined(__riscv) || defined(__arc__)
 #define DOUBLE_CONVERSION_CORRECT_DOUBLE_OPERATIONS 1
 #elif defined(__mc68000__)
 #undef DOUBLE_CONVERSION_CORRECT_DOUBLE_OPERATIONS
-- 
2.17.1

