From d1e8beda4b030a54723dad5b5ad02286786f0a2a Mon Sep 17 00:00:00 2001
From: Nobuhiro Iwamatsu <nobuhiro.iwamatsu@miraclelinux.com>
Date: Mon, 10 Dec 2018 04:48:54 +0000
Subject: [PATCH] install ptest

Signed-off-by: Nobuhiro Iwamatsu <nobuhiro.iwamatsu@miraclelinux.com>
---
 testsuite/local.mk | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/testsuite/local.mk b/testsuite/local.mk
index b4a4f5a..aaa5458 100644
--- a/testsuite/local.mk
+++ b/testsuite/local.mk
@@ -200,6 +200,11 @@ EXTRA_DIST += \
 
 # automake makes `check' depend on $(TESTS).  Declare
 # dummy targets for $(TESTS) so that make does not complain.
+install-ptest:
+	cd $(BUILDDIR); tar -c --exclude=*.o $(TESTDIR) | ( cd $(DESTDIR) && tar -xf - )
+	for i in $(EXTRA_DIST) tst-regex2.c; do install $(srcdir)/$$i $(DESTDIR)/$(TESTDIR); done
+	sed -e 's/^Makefile:/_Makefile:/' -e 's/^srcdir = \(.*\)/srcdir = ./' -e 's/bash/sh/' -i $(DESTDIR)/$(TESTDIR)/Makefile
+	for i in `grep -rl "../sed/sed" $(DESTDIR)/$(TESTDIR)`; do sed -e 's/..\/sed\/sed/sed/' -i $$i; done
 
 .PHONY: $(SEDTESTS)
 $(SEDTESTS):
-- 
2.17.1

